CONTRIBUTION.md

```diff
@@ -0,0 +1,32 @@
+# Development Contribution
+
+## Code of Conduct
+
+
+## Design
+
+
+## Getting Started
+There isn't an orchistration automation required or available. Please see each projct for instructions below.
+
+```
+$ cd cli
+$ yarn
+```
+
+### Mobile Application
+Assuming you followed the initial instructions above uild for the CLI above...
+
+```
+$ cd KukaApp
+$ yarn setup
+```
+
+#### Using Storybook
+Open `.env` and modify `STORYBOOK=true`. The default is false.
+
+Then `$ yarn cache:clear` before running either `$ yarn ios` or `$ yarn android`
+
+
+## Running Usage
+Once the application is running, please signup as there is no "development mode" or "test mode" for Authentication.
```

Comment:
Most of these instructions can be relegated to the README.md files. CONTRIBUTING.md is more about the process (PRs, issues).